File: src_java/main/java/com/github/davinkevin/transmissionrss/feeds/model/PatternMatcherProperty.java
package com.github.davinkevin.transmissionrss.feeds.model;

import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.Data;

/**
 * Created by kevin on 08/04/2018
 */
@Data
public class PatternMatcherProperty {
    private String matcher;
    private String exclude;
    private @JsonProperty("download_path") String downloadPath;
}

File: src/main/kotlin/com/github/davinkevin/transmissionrss/feeds/model/PatternMatcherProperty.kt
package com.github.davinkevin.transmissionrss.feeds.model

import com.fasterxml.jackson.annotation.JsonProperty

/**
 * Created by kevin on 08/04/2018
 */
data class PatternMatcherProperty(
    var matcher: String? = null,
    var exclude: String? = null,
    @JsonProperty("download_path") var downloadPath: String? = null
)
